bundled-guava/src/main/java/org/apache/iceberg/GuavaClasses.java

```diff
@@ -71,6 +72,7 @@ public class GuavaClasses {
     Splitter.class.getName();
     Throwables.class.getName();
     BiMap.class.getName();
+    EvictingQueue.class.getName();
```

Comment:
Do we have an easy alternative here?
Last time I have tried to add classes here, Ryan said that he tries to avoid adding further guava classes, if it is not strictly necessary.

Reply:
we can avoid it by using an array and a round-robin cursor.